src/libutil/error.cc

```diff
@@ -185,15 +185,15 @@ void printAtPos(const ErrPos & pos, std::ostream & out)
     if (pos) {
         switch (pos.origin) {
             case foFile: {
-                out << fmt(ANSI_BLUE "at " ANSI_WARNING "%s:%s" ANSI_NORMAL ":", pos.file, showErrPos(pos));
+                out << fmt(ANSI_BLUE "  at " ANSI_WARNING "%s:%s" ANSI_NORMAL ":", pos.file, showErrPos(pos));
                 break;
             }
             case foString: {
-                out << fmt(ANSI_BLUE "at " ANSI_WARNING "«string»:%s" ANSI_NORMAL ":", showErrPos(pos));
+                out << fmt(ANSI_BLUE "  at " ANSI_WARNING "«string»:%s" ANSI_NORMAL ":", showErrPos(pos));
                 break;
             }
             case foStdin: {
-                out << fmt(ANSI_BLUE "at " ANSI_WARNING "«stdin»:%s" ANSI_NORMAL ":", showErrPos(pos));
+                out << fmt(ANSI_BLUE "  at " ANSI_WARNING "«stdin»:%s" ANSI_NORMAL ":", showErrPos(pos));
```

Comment:
Still want this indent?

Reply:
I changed the format from

```
  ... error foo
  
  at location x:y
  
```

to 


```
  ... error foo
      at location x:y
  
```

The indent does just that.

Reply:
OK to make this a bit more self documenting I might do
```c++
out << fmt(ELLIPSIS_INDENT ANSI_BLUE "at "
```
where `ELLIPSIS_INDENT` is
```c++
// Enough indent to align with with the `... `
// prepended to each element of thece trace
constexpr std::string_view ELLIPSIS_INDENT = "  ";
```